DOCSP-38635 sync active source cluster (#403)
mongodb/docs-cluster-to-cluster-sync: source/quickstart.txt

  meet the minimum patch :ref:`version requirements
  <c2c-server-version-compatibility>`.

- The source cluster may remain active during sync. After it

>   performs the initial sync, ``mongosync`` continues to sync new

Reviewer: The second sentence here could imply that it doesn't sync during initial sync. I think you can just say that it can remain active during the sync because mongosync syncs new writes on the source cluster for the lifetime of the migration

Author: Fixed

> - The source cluster remains active during sync. ``mongosync``
>   syncs new writes on the source cluster for the duration of the
>   migration.

Reviewer: ```suggestion
- The source cluster can remain active during a migration until commit because ``mongosync``
  syncs writes on the source cluster for the duration of the
  migration.
```

Reviewer: Something like this. 
- The source cluster can remain active, doesn't necessarily need to
- Writes need to be stopped pre commit
- Took out the word new because it syncs existing data too

Author: Fixed

>   duration of the migration.

Reviewer: ```suggestion
  duration of the migration until commit is called.
```
Just to be extra clear
